invenio_accounts_rest/views.py

```diff
@@ -81,3 +237,67 @@ def get(self):
             ).all()
         else:
             return User.query.all()
+
+
+blueprint.add_url_rule(
+    '/roles',
+    view_func=ListRolesResource.as_view(
+        ListRolesResource.view_name
+    )
+)
+
+blueprint.add_url_rule(
+    '/roles/<string:role_id>',
+    view_func=RoleResource.as_view(
+        RoleResource.view_name
+    )
+)
+
+
+blueprint.add_url_rule(
+    '/roles/<string:role_id>/users/<string:user_id>/assign',
+    view_func=AssignRoleResource.as_view(
+        AssignRoleResource.view_name
+    )
+)
+
+
+blueprint.add_url_rule(
+    '/roles/<string:role_id>/users/<string:user_id>/unassign',
+    view_func=UnassignRoleResource.as_view(
+        UnassignRoleResource.view_name
+    )
+)
+
+
+blueprint.add_url_rule(
+    '/users/<string:user_id>/roles',
+    view_func=UserRolesListResource.as_view(
+        UserRolesListResource.view_name
+    )
+)
+
+
+blueprint.add_url_rule(
+    '/users/<string:user_id>/properties',
```

Comment:
GET '/users/<string:user_id>'
should be enough IMHO. Or is it conflicting with some other endpoint.